src/components/Lend/LoanSearch/LoanSearchFilterChips.vue

```diff
@@ -81,7 +122,7 @@ export default {
 				itemList = [...itemList, ...themeFacets];
 			}
 			// Gender
-			if (loanSearchState.gender !== null) {
+			if (loanSearchState.gender) {
```

Comment:
Does this result in showing a generic "All" chip for gender? Or is `.gender` just not present if it's not set?